html/browsers/browsing-the-web/history-traversal/resources/api-availability-1.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>API availability following history traversal - 1</title>
+<script>
+var controller = opener || parent;
+var t = controller.t;
+var assert_not_equals = controller.assert_not_equals;
+
+t.step(function() {
+  // If this document is discarded as a result of navigation, then this script
+  // will be executed a second time. The semantics this test intends to verify
+  // cannot be observed under these conditions, the discarding is not itself a
+  // violation. Silently pass the test in that case.
+  if (controller.hasNavigated) {
+    t.done();
+    return;
+  }
+
+  t.step_timeout(function() {
+    assert_not_equals(navigator, null);
```

Comment:
Maybe make this window or self.navigator to make it clear it's not a local variable.

Reply:
You got it